src/components/DashboardNavbar/DashboardNavbar.scss

```diff
@@ -0,0 +1,20 @@
+.dashboard-navbar {
+  padding-top: 10px;
+  padding-bottom: 10px;
+
+  .title {
+    font-family: Roboto;
```

Comment:
Roboto font is set for the whole page by default, I don't think that it should be specified explicitly.